components/stack-entry.js

```diff
@@ -37,6 +37,9 @@ class StackEntry extends PureLayoutComponent {
     if (landscape) {
       illustrationStyle.paddingTop = max(0, round((codeHeight - illustrationHeight) / 2));
       codeStyle.paddingTop = max(0, round((illustrationHeight - codeHeight) / 2));
+
+      illustrationStyle.display = 'table-cell';
+      codeStyle.display = 'table-cell';
```

Comment:
How about grouping these assignments per side? E.g.

```js
Object.assign(illustrationStyle, {
   display: // ...
   paddingTop: // ...
   verticalAlign: 'top' // Moved this here because it's only relevant when display is also table-cell, right?
});